.changelog/812.feature.md

```diff
@@ -0,0 +1,9 @@
+feature: Add support for runtime rofl transactions
+
+New runtime transactions:
```

Comment:
Not an ask for this PR, but we should think about adding another e2e regression suite that encompasses rofl/sapphire/consensusaccount txs and the rest of the new features being added :)

Reply:
Good idea, opened https://github.com/oasisprotocol/nexus/issues/826